### PR TITLE
Fund Buyer's children accounts

### PR DESCRIPTION
**Pull request by nicoayala**
We need a way to send ETH and WIB to buyer's children accounts so that those accounts are able to send transactions to the ethereum network and pay WIB to players.

This PR implements all the functionality needed to keep children accounts with funds.

### Tasks
~~- [ ] Periodically check account balances for both ETH and WIB amounts~~
- [x] Build, sign and send ETH transfer transactions
- [x] Build, sign and send WIB transfer transactions

### CODE REVIEW

File: buyer-api/src/facades/transferFacade.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Checks destinatary's balance and transfers funds if needed.
+ *
+ * @async
+ * @param {Object} root Buyer's root account
```
Comment (manelli) [inline buyer-api/src/facades/transferFacade.js]: I believe this param description is not necessary

File: buyer-api/src/server.js [hunk 2]
```diff
@@ -10,6 +10,18 @@ const server = () => {
     logger.info(`Buyer API listening on port ${port} and host ${host} in ${env} mode`));
 
   attachContractEventSubscribers(contractEventSubscribers, app.locals.stores);
+
+  // TODO: This is going to be replaced by the monitoring recurring function.
```
Comment (manelli) [inline buyer-api/src/server.js]: This should be removed

File: buyer-api/src/utils/contracts.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ const dataExchange = DataExchangeContract.at(config.contracts.addresses.dataExch
 
 export {
   wibcoin,
+  wibcoin as token,
```
Comment (manelli) [inline buyer-api/src/utils/contracts.js]: Why is this needed?

File: buyer-api/src/utils/index.js [hunk 4]
```diff
@@ -12,5 +12,5 @@ export {
   listLevelValues,
 } from './storage';
 export { errorHandler, asyncError, validateAddress } from './routes';
-export { wibcoin, dataExchange, DataOrderContract } from './contracts';
+export { wibcoin, token, dataExchange, DataOrderContract } from './contracts';
```
Comment (manelli) [inline buyer-api/src/utils/index.js]: Why is this needed?

File: buyer-api/src/utils/contracts.js (hunk 3 above)
Comment (nicoayala) [reply buyer-api/src/utils/contracts.js]: I know it's out of scope, but it felt bad to add more usage of the WIBToken as `wibcoin` knowing that we are changing it sooner or later. I'll revert this to preserve the scope of the PR.

File: buyer-api/src/utils/index.js (hunk 4 above)
Comment (nicoayala) [reply buyer-api/src/utils/index.js]: Same as above.
